substrate/frame/support/test/tests/construct_runtime_ui/deprecated_where_block.stderr

```diff
@@ -449,69 +449,3 @@ note: required by a bound in `Result`
    | pub enum Result<T, E> {
    |                 ^ required by this bound in `Result`
    = note: this error originates in the derive macro `self::sp_api_hidden_includes_construct_runtime::hidden_include::__private::codec::Decode` which comes from the expansion of the macro `construct_runtime` (in Nightly builds, run with -Z macro-backtrace for more info)
-
-error[E0277]: the trait bound `Runtime: Config` is not satisfied
-  --> tests/construct_runtime_ui/deprecated_where_block.rs:20:1
```

Comment:
Why did you remove this?

Reply:
This was removed by running the following command: `./scripts/update-ui-tests.sh 1.75`

Reply:
As mentioned in the comment above, it is not clear which version of rustc I should use when running `update-ui-tests.sh`. Even the bot command did not generate any diff.

Just reverted the change.

Reply:
OK that's strange. Now that you've reverted those changes, I'll run the bot command to see if it gives the proper output this time.

In general I'd recommend using the same versions as the CI for all builds and cargo commands: 1.75 and nightly-2024-01-22